test/src/test/java/test/security/realm/InMemorySecurityRealm.java

```diff
@@ -36,8 +36,21 @@
 import org.springframework.security.core.userdetails.UsernameNotFoundException;
 
 /**
- * Accept any password
- * Allow creation and removal of users
+ *  This class provides an in-memory implementation of Jenkin's SecurityRealm for
```

Comment:
```suggestion
 *  This class provides an in-memory implementation of Jenkins's {@link SecurityRealm} for
```

You resolved my previous comment correcting `Jenkin` without addressing the suggestion, please don't do that.

Wrt. the `@link`, check in IDE first to ensure this suggestion is correct.

Reply:
The @link part has been corrected with IDE specific suggestion.